src/XliffTasks/buildCrossTargeting/XliffTasks.targets

```diff
@@ -28,4 +28,42 @@
              />
   </Target>
 
+  <!--
+    Sorts the trans-unit elements in the .xlf files by the value of their id attributes.
+    Runs a sort per target framework as resource files may be conditioned on the framework.
+  -->
+  <Target Name="SortXlf">
+    <ItemGroup>
+      <_XlfTargetFramework Include="$(TargetFrameworks)" />
+    </ItemGroup>
+
+    <!--
+      These inner builds cannot be parallelized as they would race to 
+      update the same inner xlf files.
+    -->
+    <MSBuild Projects="$(MSBuildProjectFile)"
+             Targets="SortXlf"
+             Properties="TargetFramework=%(_XlfTargetFramework.Identity)"
+             />
+  </Target>
+
+  <!--
+    Checks that all trans-unit elements in the .xlf files have an up-to-date translation.
+    Runs a sort per target framework as resource files may be conditioned on the framework.
+  -->
+  <Target Name="EnsureAllResourcesTranslated">
+    <ItemGroup>
+      <_XlfTargetFramework Include="$(TargetFrameworks)" />
+    </ItemGroup>
+    
+    <!--
+      These inner builds can be run in parallel as they don't modify any files.
+    -->
+    <MSBuild Projects="$(MSBuildProjectFile)"
+             Targets="EnsureAllResourcesTranslated"
+             Properties="TargetFramework=%(_XlfTargetFramework.Identity)"
+             BuildInParallel="true"
```

Comment:
This won't actually build in parallel because the batching over _XlfTargetFramework will prevent it.

See how actual build target is parallelized:

https://github.com/Microsoft/msbuild/blob/8e015cc049b5f3f9cbd9e10ee30ae86998d934d5/src/Tasks/Microsoft.Common.CrossTargeting.targets#L67-L71

https://github.com/Microsoft/msbuild/blob/8e015cc049b5f3f9cbd9e10ee30ae86998d934d5/src/Tasks/Microsoft.Common.CrossTargeting.targets#L41-L43